src/main/scala/com/microsoft/hyperspace/index/IndexLogEntry.scala

```diff
@@ -33,14 +33,9 @@ import com.microsoft.hyperspace.HyperspaceException
 import com.microsoft.hyperspace.actions.Constants
 import com.microsoft.hyperspace.util.{PathUtils, SchemaUtils}
 
-// IndexLogEntry-specific fingerprint to be temporarily used where fingerprint is not defined.
```

Comment:
We will need to eventually implement this fingerprint. So, let's not remove this for now.

Reply:
Done

Reply:
IMHO it would be helpful for new contributors if there are comments for unused dummy code or placeholders describing the intention.